web/buildjs.sh

```diff
@@ -1,8 +1,7 @@
 #!/usr/bin/env sh
 
 mkdir -p ./build && \
-./node_modules/@babel/cli/bin/babel.js ./src/track.js --out-file ./build/track.js && \
+./node_modules/rollup/dist/bin/rollup && \
 ./node_modules/@babel/cli/bin/babel.js ./src/inline.js --out-file ./build/inline.int.js  && \
-./node_modules/uglify-js/bin/uglifyjs -O max_line_len=160  ./build/inline.int.js > ./build/inline.js &&\
-cp ./welcome.html ./build \
-&& cp ./src/track.js ./build/ #temp solution since, track.js with babel doesn't work
\ No newline at end of file
+./node_modules/uglify-js/bin/uglifyjs -O max_line_len=160  ./build/inline.int.js > ./build/inline.js && \
```

Comment:
Should we add compiling of inline.js to rollup.config.js as well? (for having one way of compiling js code) @dubrowsky